static/src/stylesheets/base/_links.scss

```diff
@@ -12,4 +12,9 @@ a,
         color: $news-accent;
         text-decoration: none;
     }
+    // Improve readability of link text in figcaptions that are on dark backgrounds (issue #12606)
+    .tonal--tone-media .caption--img & {
```

Comment:
I think tone related styles should be in the tone scss file and not base? https://github.com/guardian/frontend/blob/master/static/src/stylesheets/module/content/tones/_tone-media.scss